src/kudu/consensus/quorum_util.cc

```diff
@@ -295,13 +296,23 @@ namespace {
 typedef map<string, pair<RaftPeerPB, RaftPeerPB>> PeerInfoMap;
 
 bool DiffPeers(const PeerInfoMap& peer_infos,
-               vector<string>* change_strs) {
+               vector<string>* change_strs,
+               vector<string>* evicted_peers) {
```

Comment:
you already allow for nullptr here. Do you want to make nullptr as default param, so that the test
file does not have to change?